ci-operator/step-registry/openstack/conf/generateconfig/openstack-conf-generateconfig-commands.sh

```diff
@@ -8,40 +8,70 @@ CLUSTER_NAME=$(<"${SHARED_DIR}/CLUSTER_NAME")
 OPENSTACK_EXTERNAL_NETWORK="${OPENSTACK_EXTERNAL_NETWORK:-$(<"${SHARED_DIR}/OPENSTACK_EXTERNAL_NETWORK")}"
 OPENSTACK_COMPUTE_FLAVOR="${OPENSTACK_COMPUTE_FLAVOR:-$(<"${SHARED_DIR}/OPENSTACK_COMPUTE_FLAVOR")}"
 
-LB_FIP_IP=$(<"${SHARED_DIR}"/LB_FIP_IP)
-INGRESS_FIP_IP=$(<"${SHARED_DIR}"/INGRESS_FIP_IP)
+if [[ "${CONFIG_TYPE}" == "fipless" ]]; then
+  API_VIP=$(<"${SHARED_DIR}"/API_VIP)
+  INGRESS_VIP=$(<"${SHARED_DIR}"/INGRESS_VIP)
+else
+  LB_FIP_IP=$(<"${SHARED_DIR}"/LB_FIP_IP)
+  INGRESS_FIP_IP=$(<"${SHARED_DIR}"/INGRESS_FIP_IP)
+fi
 
 PULL_SECRET=$(<"${CLUSTER_PROFILE_DIR}"/pull-secret)
 SSH_PUB_KEY=$(<"${CLUSTER_PROFILE_DIR}"/ssh-publickey)
 
 CONFIG="${SHARED_DIR}/install-config.yaml"
-if [[ "${CONFIG_TYPE}" == "minimal" ]]; then
+
+case "$CONFIG_TYPE" in
```

Comment:
not in?

Reply:
ok I'll change it in the next iteration.

Reply:
I agree it's a bit confusing to use a switch statement like that, but IIUC the code is functionally correct: it exits with an error unless you use CONFIG_TYPE minimal or fipless.

Reply:
I'm torn... I find the following more readable but less maintainable:
```sh
if [[ "$CONFIG_TYPE" != "minimal" || "$CONFIG_TYPE" != "fipless"]]; then
    echo "No valid install config type specified. Please check CONFIG_TYPE"
    exit 1
fi
```
WDYT?

Reply:
I would be in favor of keeping the `case` which is easier to read, in my taste.